cf/aggregate.py

```diff
@@ -258,10 +261,19 @@ def __init__(
             aux_coord.nc_set_variable(prop)
             aux_coord.id = prop
 
+            if _copy:
+                # Copy the field, as we're about to change it.
+                f = f.copy()
+                self.field = f
+                _copy = False
```

Comment:
Is this line strictly necessary? `_copy` doesn't seem to be used again in the constructor method and it doesn't seem to make any difference from what I can see with some `print` statements, but I could be missing something.

Reply:
Hi Sadie, setting `_copy = False` is to prevent the field from being copied multiple times in the event that `dimension` has more than one element. We initialise it to the "copy" keyword because we don't mind about changing the input fields in place if the user has specified `cf.aggregate(..., copy=False)`.

Reply:
Aha, I see now. Thanks for clarifying David. All good, approving now!